codemods/index.js

```diff
@@ -1,5 +1,308 @@
+import isWhitespace from './is-whitespace/index.js';
```

Comment:
Hm, I think something might have gone wrong while rebasing, this file looks different in the main branch: https://github.com/thepassle/module-replacements-codemods/blob/main/codemods/index.js

Reply:
Oops, I fixed it. Thanks for checking